MantidPlot/src/Mantid/MantidTable.cpp

```diff
@@ -423,6 +429,19 @@ void MantidTable::sortColumns(const QStringList &s, int type, int order,
   }
 }
 
+/** Set the plot type on the workspace for each selected column
+ *
+ * @param plotType :: the plot type to set the selected columns to.
+ */
+void MantidTable::setPlotTypeForSelectedColumns(int plotType) {
+  const QStringList list = selectedColumns();
+  for (int i = 0; i < static_cast<int>(list.count()); i++) {
```

Comment:
I think this can be converted to a range-based for loop?

Reply:
Yes. You're right. 👍